science/ngspice/Portfile

```
@@ -3,12 +3,11 @@
 PortSystem      1.0
 
 name            ngspice
-version         36
+version         43
 revision        0
 license         BSD
 categories      science cad
-maintainers     nomaintainer
-platforms       darwin
+maintainers     {bpdegnan @bpdegnan}
```

You don't have the email address `bpdegnan` at `macports.org` so the first part of this clause is wrong. Please list your real email address there in the obfuscated format `host:user`, for example:

```suggestion
maintainers     {example.com:bpdegnan @bpdegnan}
```

Is it ok if other people make minor changes to this port without asking you? If so, please add `openmaintainer` as well:

```suggestion
maintainers     {example.com:bpdegnan @bpdegnan} openmaintainer
```